tests/e2e/state.go

```diff
@@ -465,6 +467,29 @@ func (tr Commands) GetProposal(chain ChainID, proposal uint) Proposal {
 			Title:       title,
 			Description: description,
 		}
+	case "/interchain_security.ccv.provider.v1.MsgUpdateConsumer":
+		consumerId := rawContent.Get("consumer_id").String()
+		consumerChainId := ChainID("")
+		for _, chainCfg := range tr.chainConfigs {
+			if chainCfg.ConsumerId == e2e.ConsumerID(consumerId) {
+				consumerChainId = chainCfg.ChainId
```

Comment:
Shall we `break` here for clarity if we do not have more than one chain config with the same `consumerId`?